spec/test-suite.json

```diff
@@ -0,0 +1,71 @@
+{
+    "suite":
+    {
+        "zmanim":
+        [
+            {
+                "setup":
+                {
+                    "calculator":
+                    {
+                        "name": "SunTimes",
+                        "zmanimInformation":
+                        {
+                            "name": "",
+                            "url": ""
+                        },
+                        "information":
+                        {
+                            "name": "US Naval Observatory's Almanac for Computer algorithm",
+                            "url": "http://aa.usno.navy.mil/publications/docs/asa.php"
+                        }
+                    },
+                    "date": "2010-04-02",
+                    "location":
+                    {
+                        "name": "Lakewood, NJ",
+                        "lat": 40.09596,
+                        "long": -74.22213,
+                        "elevation": 0,
+                        "timeZone":
+                        {
+                            "offset": -5,
+                            "zoneInfoName": "America/New_York"
+                        }
+                    }
+                },
+                "tests":
+                [
+                    {
+                        "method": "sunset",
+                        "description": "Gets the exact sunset time",
+                        "extra":
+                        {
+                            "edgeCase": false,
+                            "information": ""
+                        },
+                        "expected":
+                        {
+                            "utc": "2010-04-02T12:22:3:710Z",
+                            "local": "2010-04-02T19:22:3:710-05:00"
```

Comment:
I'm all for including local as a nice-to-have, but it could be tricky for implementations that weren't expecting to have to handle this.

Reply:
I have mixed feelings about tz and local. I would Make tz optional and only use it when testing something with tz and removing local/UTC and just having one expected value. The main downside is readability, and when someone creates a new test case, they need to make the conversation.

Reply:
This was fixed